src/components/create-study-form.js

```diff
@@ -208,10 +199,9 @@ export const CreateStudyForm = () => {
 
     return (
         <div>
-            <Button variant="contained" color="primary" className={classes.addButton} onClick={() => handleClickOpenDialog() }>
-                <AddIcon className={classes.addIcon}/>
-                <FormattedMessage id="newStudy"/>
```

Comment:
You forgot to remove it from the translation resource files?

Reply:
Indeed! I'm on it